specs/interop/dependency-set.md

```diff
@@ -6,6 +6,9 @@
 
 - [Chain ID](#chain-id)
 - [Updating the Dependency Set](#updating-the-dependency-set)
+- [Architecture](#architecture)
+  - [L2 DependencyManager](#l2-dependencymanager)
+  - [L1 SuperchainConfigInterop](#l1-superchainconfiginterop)
```

Comment:
small nit: we will be merging `SuperchainConfigInterop` into the mainline `SuperchainConfig` as soon as we know interop is the next release. Its better to just call it by the base name `SuperchainConfig` so that we don't go out of date here